addons/website_slides/models/slide_slide.py

```diff
@@ -108,7 +108,7 @@ class Slide(models.Model):
     }
     _order = 'sequence asc, is_category asc, id asc'
 
-    YOUTUBE_VIDEO_ID_REGEX = r'^.*(youtu.be\/|v\/|u\/\w\/|embed\/|watch\?v=|&v=)([^#&?]*).*'
+    YOUTUBE_VIDEO_ID_REGEX = r'^(?:(?:https?:)?//)?(?:www\.|m\.)?(?:youtube(-nocookie)?\.com/(?:embed/|v/|shorts/|live/|watch\?v=|watch\?.+&v=)|youtu\.be/)((?:\w|-){11})\S*$'
```

Comment:
Side note: We should one day stop using regex to parse those urls because there are just too much Youtube link variants and some of the groups we have are incompatible with each other (for dailymotion notably). The expression also becomes complicated...

Reply:
Thanks for the review :) I agree the regex just get messier and messier, in this case I'm trying to make it consistent with 'web_editor' regex they're practically the same but also accepting the urls as "https://m.youtube.com/..." which is currently not included, maybe we could make them the same regex while we still use regex, or just update only this for this time 🤔 (the mobile url is already in the test)

Reply:
As you are targeting 16.0, I would go for the first option: I would adapt the `addons/web_editor/tools.py` regex to add the support of mobile and live stream links. Then, I would reuse the same regex in `/website_slides` for consistency.

Rational: If we have tickets because we do not recognize those urls, we will probably have similar tickets in the `web_editor`. I assume we should also be able to embed live streams in the editor. It uses the same Youtube player, right? So, why shouldn't we be allowed to paste mobile or live stream links in the editor?

Reply:
Yes, that makes sense so we are consistent in every place possible, I specified in the commit, editor can take livestreams for example, but not if you use the link generated from Youtube when using the share option, so after this it should work for most of the urls.